src/utils/schema.js

```diff
@@ -10,6 +10,7 @@
  * src/utils/schema.js
  */
 const Joi = require('joi');
+const urlRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-]*)?\??(?:[\-\+=&;%@\.\w]*)#?(?:[\.\!\/\\\w]*))?)/g; // jscs:ignore maximumLineLength
```

Comment:
I think in July 2017, we decided to use the uri validator in Joi instead of regex. I think there is a story for this.